lib/css/components/_stacks-tables.less

```diff
@@ -315,6 +314,23 @@
             opacity: 0.3;
         }
     }
+
+    //  $$  Responsive
+    //  ------------------------------------------------------------------------
+    &.s-table__responsive {
```

Comment:
When applying a background color to `s-table` I also discovered a side effect of applying `display: block` here... struggling to get the non overflowed table to take up 100% of its parent's width. Tables 😖